core/jazz_environment-event-handler/index.js

```diff
@@ -651,7 +877,14 @@ const exportable = {
   processServiceDetails,
   manageProcessItem,
   processRequest,
-  getSvcPayload
+  getSvcPayload,
+  addSafe,
+  removeSafe,
+  createSafe,
+  deleteSafe,
+  getAdmins,
```

Comment:
Since all these are related to safe (t-vault), rename `getAdmins` to `getSafeAdmins` and `getEnvDetails` to `getEnvSafeDetails` so the purpose is clear.

Reply:
As discussed, since getAdmins and getEnvDetails are not tvault specific apis, hence moved out of Tvault obj in config and addressed